docs/source/tutorials/Build_New_Project_on_LiBai.md

```diff
@@ -0,0 +1,229 @@
+
+The complete file structure of the project:
+
+```
+projects/my_project
+├── configs
+│   └── config_custom.py
+│   └── ...
+├── dataset
+│   ├── custom_dataset.py
+│   └── ...
+├── modeling
+│   ├── custom_model.py
+│   └── ...
+├── README.md
+```
+
+Starting a new project based on LiBai step by step:
+
+1. Prepare a config file(such as [config.py](https://github.com/Oneflow-Inc/libai/blob/main/projects/QQP/configs/config_qqp.py)) which contains:
+    - This file should be independent of the default config in LiBai.
+    - The relevant parameters of the task.
+    - Defining related Class, such as `Model`, `Optimizer`, `Scheduler`, `Dataset`.
+    - You can inherit the default config in `configs/common` and rewrite it, which can greatly reduce the workload.
+    - Related class defined with LazyCall which returns a dict instead of calling the object.
+
+2. Prepare a model file(such as [model.py](https://github.com/Oneflow-Inc/libai/blob/main/projects/QQP/modeling/model.py)) which contains:
+    - Build related models in this file, the construction method is similar to OneFlow.
+    - Because Libai will set up a static diagram by default, the calculation of loss needs to be inside the model.
+    - The function `forward` must return a dict.
+    - When defining a tensor in the model, you need to use `to_global`, turn tensor into a global pattern.
+    - When defining layers, you can import them directly from `libai.layers`, because it have already pre-defined the SBP signature.
+
+3. Prepare a dataset file(such as [dataset.py](https://github.com/Oneflow-Inc/libai/tree/main/projects/QQP/dataset)) which contains:
+    - Build `Dataset` in this file, the construction method is similar to OneFlow.
+    - The difference is that we need to use `DistTensorData` and `Instance`.
+    - The shape of each batch must be global.
+    - The return `key`  
+    - In `__getitem__` function, the `key` returned by the method must be global with the parameter name of the 'forward' function in the 'model'.
+
+
+## Main Function Entry
+[tools/train_net.py](https://github.com/Oneflow-Inc/libai/blob/main/tools/train_net.py) is the default main function entry provided in LiBai.
+
+
+## Build Config
+The `config.py` in LiBai is special, which takes the form of lazyconfig and will be saved as `.yaml` at runtime.
+The following describes the complete `config.py` and how to inherit the config in LiBai.
+
+First, config has several necessary fields:
+- `train`: It contains training related parameters and is a dict type.
+- `model`: Model used by the task, specify the generation method in the file, due to the characteristics of lazycall, the model     will be generated at runtime.
+- `optim`: Optimizer related. Default to AdamW.
+- `lr_scheduler`: Related to learning rate, Default to warmup cosine decay lr-scheduler.
+- `graph`: Import directly, and the model will be automatically converted to graph during operation.
```

Comment:
convert to nn.Graph mode for training and evaluation, please refer to [nn.Graph docs](https://oneflow.readthedocs.io/en/master/graph.html) for more details